### ISSUE TITLE
change the output of zos container 5 zerotierlist

### ISSUE THREAD

**Issue by Dina-Abd-Elrahman** [threefoldtecharchive/zosctl#35]
need to change the output of this command 

- zos container 5 zerotierlist

```
[{"allowDefault":false,"allowGlobal":false,"allowManaged":true,"assignedAddresses":["fc2e:9ff1:7461:203f:5e0b:0000:0000:0001/40","10.244.190.201/16"],"bridge":false,"broadcastEnabled":true,"dhcp":false,"id":"9bee8941b5717835","mac":"36:19:51:8a:1f:82","mtu":2800,"name":"tfgrid_public","netconfRevision":3,"nwid":"9bee8941b5717835","portDeviceName":"zt3jn7qoma","portError":0,"routes":[{"flags":0,"metric":0,"target":"10.244.0.0/16","via":null}],"status":"OK","type":"PUBLIC"}]
```

to be printed like this :

```
[
  {
    "allowDefault": false,
    "allowGlobal": false,
    "allowManaged": true,
    "assignedAddresses": [
      "fc2e:9ff1:7461:203f:5e0b:0000:0000:0001/40",
      "10.244.190.201/16"
    ],
    "bridge": false,
    "broadcastEnabled": true,
    "dhcp": false,
    "id": "9bee8941b5717835",
    "mac": "36:19:51:8a:1f:82",
    "mtu": 2800,
    "name": "tfgrid_public",
    "netconfRevision": 3,
    "nwid": "9bee8941b5717835",
    "portDeviceName": "zt3jn7qoma",
    "portError": 0,
    "routes": [
      {
        "flags": 0,
        "metric": 0,
        "target": "10.244.0.0/16",
        "via": null
      }
    ],
    "status": "OK",
    "type": "PUBLIC"
  }
]

```

**Comment by xmonader** [comment]
Fixed in 43bb7eb8110175deeaf2127fb9752feb4b6baf94

**Comment by Dina-Abd-Elrahman** [comment]
# ZOS Version:
43bb7eb8110175deeaf2127fb9752feb4b6baf94

# Verification Steps:
- ./zos container 4 zerotierlist 

```
[
  {
    "allowDefault": false,
    "allowGlobal": false,
    "allowManaged": true,
    "assignedAddresses": [
      "fc2e:9ff1:7465:119b:a4f1:0000:0000:0001/40",
      "10.244.217.195/16"
    ],
    "bridge": false,
    "broadcastEnabled": true,
    "dhcp": false,
    "id": "9bee8941b5717835",
    "mac": "36:1d:60:2e:e5:78",
    "mtu": 2800,
    "name": "tfgrid_public",
    "netconfRevision": 3,
    "nwid": "9bee8941b5717835",
    "portDeviceName": "zt3jn7qoma",
    "portError": 0,
    "routes": [
      {
        "flags": 0,
        "metric": 0,
        "target": "10.244.0.0/16",
        "via": null
      }
    ],
    "status": "OK",
    "type": "PUBLIC"
  }
]
```
- ./zos container 4 zerotierinfo 

```
{
  "address": "65119ba4f1",
  "clock": 1539697489053,
  "config": {
    "physical": null,
    "settings": {
      "allowTcpFallbackRelay": true,
      "portMappingEnabled": true,
      "primaryPort": 64745,
      "softwareUpdate": "disable",
      "softwareUpdateChannel": "release"
    }
  },
  "online": true,
  "planetWorldId": 149604618,
  "planetWorldTimestamp": 1532555817048,
  "publicIdentity": "65119ba4f1:0:42afbb6cc69f552566e5365b92048ec2f39883f18ca6b637b984367807b1087d910f21586c3a7fd3acd0dc8689da4267d10eed3e3b27b6831fb27e116946f006",
  "secretIdentity": "65119ba4f1:0:42afbb6cc69f552566e5365b92048ec2f39883f18ca6b637b984367807b1087d910f21586c3a7fd3acd0dc8689da4267d10eed3e3b27b6831fb27e116946f006:862deea57cb258d27b08ad8b43b230a81a6615a02a39db43e70333dbd0343612102ae18ae858bb5b3eacbc747a686ec325cf34026e2709813acf585ef980b92a",
  "tcpFallbackActive": false,
  "version": "1.2.10",
  "versionBuild": 0,
  "versionMajor": 1,
  "versionMinor": 2,
  "versionRev": 10
}
```
